operational/azure/azuread_group_sync/README.md

```diff
@@ -0,0 +1,75 @@
+# AzureAD Group Sync
+
+## What it does
+
+This policy collects groups and their members from AzureAD and synchronizes them to the Flexera Cloud Management Platform.
+
+## Functional Details
+
+This policy leverages the Azure AD Graph API to collect groups and their members based on a filter prefix. It then compares them to the groups and members that currently exist in the Flexera CMP and updates the CMP groups to reflect the current membership in AzureAD.
+
+The policy will recurse through all child groups and flatten their membership in the matching parent CMP group.
+
+If users do not exist in the CMP, they are created using the information gathered from AzureAD.
+
+You can elect to automatically remove users from your organization that are no longer members of any groups by setting selecting the `Remove Users?` parameter.
+
+## Input Parameters
+
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Prefix to filter groups on* - Uses the `startswith()` odata query to filter AzureAD groups based on a prefix
+  - Example: `FLEXERA_CMP_`
+- *Default Phone Number for New Users* - Use this phone number if the user doesn't have one set in AzureAD.
+  - The policy looks for a value in the first item of `businessPhones` attribute of the AzureAD user account.
+- *Default Company Name for New Users* - Use this Company Name if the user doesn't have one set in AzureAD
+  - The policy looks for a value in the `companyName` attribute of the AzureAD user account.
+- *Identity Provider Href* - The Href for the Identity Provider to associate to new users
+  - Can be retrieved by copying the link to edit your SSO: ht&#8203;tps://us-3.rightscale.com/global/enterprises/54321/edit_sso?identity_provider_id=**123** and grabbing the ID value at the end, or via the [RightScale Cloud Management API](https://reference.rightscale.com/api1.5/resources/ResourceIdentityProviders.html#index)  
```

Comment:
should you use this link?

Reply:
It's "fake" but I can replace it with a token.